catalyst/contrib/__main__.py

```diff
@@ -2,7 +2,7 @@
 from collections import OrderedDict
 import logging
 
-from catalyst.contrib.scripts import find_thresholds
+from catalyst.contrib.scripts import collect_env, find_thresholds
```

Comment:
Have you check, that `catalyst-contrib --collect-env` works correctly? should we add this test to `catalyst/bin/tests`?